components/modelregistry/modelregistry.go

```diff
@@ -84,12 +92,19 @@ func (m *ModelRegistry) ReconcileComponent(ctx context.Context, cli client.Clien
 			}
 		}
 
-		// Create odh-model-registries namespace
+		// Create model registries namespace
 		// We do not delete this namespace even when ModelRegistry is Removed or when operator is uninstalled.
-		_, err := cluster.CreateNamespace(ctx, cli, "odh-model-registries")
+		ns, err := cluster.CreateNamespace(ctx, cli, ModelRegistriesNamespace)
+		if err != nil {
+			return err
+		}
+		l.Info("created model registry namespace", "namespace", ModelRegistriesNamespace)
+		// create servicemeshmember here, for now until post MVP solution
+		err = enrollToServiceMesh(ctx, cli, dscispec, ns)
 		if err != nil {
 			return err
 		}
```

Comment:
maybe i missed something but if you are creating SMM here, by not use FTer to create namespace and use template?

Reply:
@zdtsw I'm not sure I understand the question. What's FTer?

Reply:
something like this ( did not test the snippet ) 

modelRegErr := registry.Add(feature.Define("modelregistry-in-mesh").
				Manifests(
					manifest.Location(Resources.Location).
						Include(
							path.Join(Resources.ServiceMeshDir, "servicemesh-member.tmpl.yaml"),
						),
				).
				Managed().
                PreConditions(
                         feature.CreateNamespaceIfNotExists(ModelRegistriesNamespace),
                ).
				WithData(
					servicemesh.FeatureData.ControlPlane.Define(dscispec).AsAction(),
				).
			)


then we can get rid of `CreateNamespace()`

Reply:
Ok, a feature. The servicemeshmember creation is actually not a long term solution in modelregistry.go and going to be replaced with something from the servicemesh team post model registry MVP release. That solution will work with a custom label on the namespace. 
Hence, the simple solution for now.

Reply:
> The servicemeshmember creation is actually not a long term solution in modelregistry.go and going to be replaced with something from the servicemesh team post model registry MVP release. That solution will work with a custom label on the namespace.

This decision has not been finalized yet, so it's unclear where this conclusion comes from. While a migration to OSSM v3 will eventually be necessary, it will be managed uniformly across all onboarded components when the time comes.